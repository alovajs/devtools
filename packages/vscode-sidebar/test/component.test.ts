import { mount } from '@vue/test-utils'
import HelloWorld from '../src/components/HelloWorld.vue'

describe('component TheCounter.vue', () => {
  it('should render', () => {
    const wrapper = mount(HelloWorld, { props: { msg: 'Hello' } })
    expect(wrapper.text()).toContain('Hello')
    expect(wrapper.html()).toMatchSnapshot()
  })
})
