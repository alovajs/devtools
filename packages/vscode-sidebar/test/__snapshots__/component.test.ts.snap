// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`component TheCounter.vue > should render 1`] = `
"<div data-v-e17ea971="" class="greetings">
  <h1 data-v-e17ea971="" class="green">Hello</h1>
  <h3 data-v-e17ea971=""> Youâ€™ve successfully created a project with <a data-v-e17ea971="" href="https://vitejs.dev/" target="_blank" rel="noopener">Vite</a> + <a data-v-e17ea971="" href="https://vuejs.org/" target="_blank" rel="noopener">Vue 3</a>. What's next? </h3>
</div>"
`;
